charts/posthog/templates/web-hpa.yaml

```diff
@@ -2,12 +2,7 @@
 apiVersion: autoscaling/v1
 kind: HorizontalPodAutoscaler
 metadata:
-  name: {{ template "posthog.fullname" . }}-web
```

Comment:
the `name` attribute was removed, which causes erros on upgrade if hpa is enabaled

Reply:
It should be fixed now @Legion2 , sorry for the issue!